frontend/public/kubevirt/components/dashboards/dashboards.jsx

```diff
@@ -2,6 +2,49 @@ import React from 'react';
 import { ClusterOverview } from '../cluster/cluster-overview';
```

Comment:
When refactoring, I would split content of this dashboard.jsx to multiple files:
- index.jsx for re-export
- utils.jsx (or similar) for functions
- dashboards.jsx for the component

Imports in other files should be updated accordingly.

Reply:
done